flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/join/temporal/LegacyTemporalProcessTimeJoinOperator.java

```diff
@@ -73,6 +73,7 @@ public void open() throws Exception {
 		this.outRow = new JoinedRowData();
 		// consider watermark from left stream only.
 		super.processWatermark2(Watermark.MAX_WATERMARK);
+		super.open();
```

Comment:
move function call of super to the beginning

Reply:
and I believe `TemporalRowTimeJoinOperator.java` exists same problem, could you help fix?

Reply:
fixed.